cmd/image.go

```diff
@@ -36,18 +36,20 @@ var targetPlatform string
 var tag string
 var artifactID string
 var region string
+var cacheFrom string
 
 func init() {
 	rootCmd.AddCommand(imageCmd)
 	imageCmd.AddCommand(imagePushCmd)
 	imagePushCmd.Flags().StringVarP(&dockerBuildContext, "build-context", "b", ".", "Path to the directory to build the image from")
 	imagePushCmd.Flags().StringVarP(&dockerfileName, "dockerfile", "f", "Dockerfile", "Name of the dockerfile to build from if you have named it anything other than Dockerfile")
 	imagePushCmd.Flags().StringVarP(&tag, "image-tag", "t", "latest", "Unique identifier for this version of the image")
-	imagePushCmd.Flags().StringVarP(&artifactID, "artifact", "a", "", "Massdriver ID of the artifact used to create the repository and generate repository credentials.")
+	imagePushCmd.Flags().StringVarP(&artifactID, "artifact", "a", "", "Massdriver ID of the artifact used to create the repository and generate repository credentials")
 	_ = imagePushCmd.MarkFlagRequired("artifact")
 	imagePushCmd.Flags().StringVarP(&region, "region", "r", "", "Cloud region to push the image to")
 	_ = imagePushCmd.MarkFlagRequired("region")
 	imagePushCmd.Flags().StringVarP(&targetPlatform, "platform", "p", "linux/amd64", "")
+	imagePushCmd.Flags().StringVarP(&cacheFrom, "cache-from", "c", "", "Folder containing images used for caching")
```

Comment:
I think this should be setup as a `StringArrayVarP` (along with the value holder) since the underlying CacheFrom is a `[]string`. 
The description should be updated to indicate it's a list of images to use as a cache, not a dir. 

From docker source `// CacheFrom specifies images that are used for matching cache...`

Reply:
> I think this should be setup as a StringArrayVarP (along with the value holder) since the underlying CacheFrom is a []string.

I'm not convinced. `Tag` is an underlying `[]string` but we only expose a flag for a single tag right now.

Reply:
Fixed description.

Reply:
Is there a linear issue for this? 

I don't follow the `Tags` to `cache` link. `cache-from` is a separate arg that docker build can use to pull images and use them as a cache. The tag for the cache image is included when specifying the image i.e. `docker build --cache-from mycool/image:1.1.1` 

However, the image had to have been built with the flag to allow it to be a cache in the first place. https://docs.docker.com/engine/reference/commandline/build/#cache-from

Reply:
I have seen a few examples on GHA of the use of docker save to build the cache for GHA. That is what we are going to attempt here. Maybe we should take this opportunity to expand tags to an array as well (assuming we can do it in a BC manner)

Reply:
Discussion about whether that should be an array or not seems moot until we can get something working.

Reply:
https://github.com/massdriver-cloud/mass/blob/238f50fd4d26519a1c7973875ee05aaf303aea8f/internal/commands/image/docker_client.go#L49